py/selenium/webdriver/support/expected_conditions.py

```diff
@@ -514,7 +513,7 @@ def any_of_condition(driver: D):
 def all_of(
     *expected_conditions: Callable[[D], Union[T, Literal[False]]]
 ) -> Callable[[D], Union[List[T], Literal[False]]]:
-    """An expectation that all of multiple expected conditions is true.
+    """An expectation that all multiple expected conditions is true.
```

Comment:
This is still grammatically incorrect and confusing.

I suggest:

An expectation that all of the expected conditions provided are true

Reply:
It's good to get rid of "all of" as that is redundant, but agreed that "all the expected conditions provided are true" is more clear.